docs/GUIDE.md

```diff
@@ -28,41 +28,52 @@ The Plutarch guide is your one stop shop for getting up to speed on Plutarch!
 - [Common Extensions and GHC options](./RUN.md#common-extensions-and-ghc-options)
 - [Evaluation](./RUN.md#evaluation)
 
-## Syntax
+## Introduction and Basic Syntax
 
-- [Constants](./SYNTAX.md#constants)
-  - [Static building with `pconstant` and `pconstantData`](./SYNTAX.md#static-building-with-pconstant-and-pconstantdata)
-  - [Dynamic building with `pcon`](./SYNTAX.md#dynamic-building-with-pcon)
-  - [Overloaded literals](./SYNTAX.md#overloaded-literals)
-  - [Miscellaneous](./SYNTAX.md#miscellaneous)
-- [Lambdas](./SYNTAX.md#lambdas)
-- [Delayed terms and Forcing](./SYNTAX.md#delayed-terms-and-forcing)
+This section serves as a introduction to Plutarch's basic concepts and core syntactic constructs. It will help build a mental model of Plutarch, but is insufficient to write production-ready code.
 
-## Usage
-
-- [Applying functions](./USAGE.md#applying-functions)
-- [Conditionals](./USAGE.md#conditionals)
-- [Recursion](./USAGE.md#recursion)
-- [Do syntax with `TermCont`](./USAGE.md#do-syntax-with-termcont)
-- [Do syntax with `QualifiedDo` and `Plutarch.Monadic`](./USAGE.md#do-syntax-with-qualifieddo-and-plutarchmonadic)
-- [Deriving typeclasses for `newtype`s](./USAGE.md#deriving-typeclasses-for-newtypes)
-- [Deriving typeclasses with generics](./USAGE.md#deriving-typeclasses-with-generics)
+- [Overview](#overview)
```

Comment:
could you update these hyperlinks please?

Reply:
Yep, fixed a bunch of these. I think I got them all.